packages/dataviews/src/dataviews-layouts/list/style.scss

```diff
@@ -114,12 +114,12 @@ ul.dataviews-view-list {
 			&::before {
 				position: absolute;
 				content: "";
-				top: calc(var(--wp-admin-border-width-focus) + 1px);
+				top: var(--wp-admin-border-width-focus);
```

Comment:
🤔 Was this change intended for another PR?

Reply:
This one is tiny visual bug I observed while testing. Figured it was okay to include here, but can also separate it out if y'all prefer.

Reply:
Up to you, really. I'd prefer a separate PR, but no strong feelings either way, so leaving to your judgment.